style.css

```diff
@@ -3,5 +3,5 @@ main {
 }
 
 h1 {
-  color: orange;
+  color: blue;

```

Comment:
In style.css

1. Kindly use the required color as it was instructed, it makes your project clean.

Reply:
> In style.css
> 
> 1. Kindly use the required color as it was instructed, it makes your project clean.

- [ ] You did well 😃 by using the polite reply, but as you see here the reason for the change is not clear to the student, so kindly 😊 comments should be clear and easy to understand. For Example:

- Instead of the required color as was instructed, it makes your project clean you can tell the student that too kindly uses the correct color as this makes the project will be similar to the original design as it will be nice for the visitors 😃